### PR TITLE
Add LICENSE.md + CONTRIBUTORS.md + remark tooling

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/encoding-down/issues/56

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,10 +118,10 @@ db.put('example', 42, function (err) {
 
 ### `const db = require('encoding-down')(db[, options])`
 
-* `db` must be an [`abstract-leveldown`] compliant store
-* `options` are passed to [`level-codec`]:
-  - `keyEncoding`: encoding to use for keys
-  - `valueEncoding`: encoding to use for values
+-   `db` must be an [`abstract-leveldown`] compliant store
+-   `options` are passed to [`level-codec`]&#x3A;
```
Comment (vweevers) [inline README.md]: Huh, remark encodes a colon?

Comment (ralphtheninja) [reply README.md]: That was odd indeed

Comment (ralphtheninja) [reply README.md]: I can remove the colon and re-render

Comment (vweevers) [reply README.md]: Nah, the colon makes sense here. We can live with an encoded thing in `encoding-down` :)

**Comment by ralphtheninja** [comment]
Just noticed that if you run `remark` again, it encodes the encoded colon _again_:

screenshot from 2018-06-22 13-52-57

Which gives the following result:

screenshot from 2018-06-22 13-53-50

**Comment by ralphtheninja** [comment]
So we should probably remove the colon, or do some other trickery

**Comment by vweevers** [comment]
Let's see if we can find the responsible module in the remark jungle ;)

**Comment by ralphtheninja** [comment]
> Let's see if we can find the responsible module in the remark jungle ;)

That's even better. It smells like a :bug:

**Pull request merged**